src/laser/Laser.H

```diff
@@ -33,12 +33,15 @@ public:
     amrex::Real m_a0 {0.}; /**< Laser peak normalized amplitude */
     amrex::Real m_w0 {0.}; /**< Laser waist */
     amrex::Real m_CEP {0.}; /**< Laser carrier-envelope phase (CEP) */
-    /** Propagation angle of the pulse in the yz plane (0 is the along the z axis) */
+    /** Propagation angle of the pulse in the yz plane (0 is along the z axis) */
     amrex::Real m_propagation_angle_yz {0.};
-    /**Pulse front tilt angle of the pulse in yz plane (pi/2 is no PFT)*/
-    amrex::Real m_PFT_yz {MathConst::pi/2.0};
     amrex::Real m_L0 {0.}; /**< Laser length (HW 1/e in amplitude) */
     amrex::Real m_tau {0.}; /**< Laser duration (HW 1/e in amplitude) */
+    /** Time stretching factors */
+    amrex::Real m_STC_theta_xy {0.};
+    amrex::Real m_zeta {0.};
+    amrex::Real m_beta {0.};
+    amrex::Real m_phi2 {0.};
```

Comment:
Please add description for each parameter separately. Can be on the same line, see above.